tensorboard/webapp/widgets/card_fob/card_fob_controller_test.ts

```diff
@@ -96,7 +96,7 @@ describe('card_fob_controller', () => {
   function createComponent(input: {
     axisDirection?: AxisDirection;
     timeSelection: TimeSelection;
-    enableRangeSelection: boolean;
+    enableRangeSelection?: boolean;
```

Comment:
I think the value passed to the component should be set to false on line 156 if this is undefined.

Reply:
I've gone ahead and done that.